src/api/ratelimits.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Rate Limits
+sidebar_order: 5
+---
+
+Sentry rate limits every API request made to prevent abuse and resource overuse. The limit is applied to each caller per endpoint.
+
+The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected.
+
+While there is a default value, each endpoint can have their own maximum number of requests and window size. You can track your rate limit usage by looking at special headers in the response
+
+## Headers
+
+Every API request response will include the following headers:
+
+- `X-Sentry-Rate-Limit-Limit`
```

Comment:
@AniketDas-Tekky why 2 `Limit`? Im assuming its a typo

Reply:
not a type, "Rate Limit" is the feature, while the 2nd Limit is the maximum number of request.
Feels weird to me too, but it's apparently pretty standard.